fastembed/common/model_management.py

```diff
@@ -117,7 +117,14 @@ def download_files_from_huggingface(
 
         return snapshot_download(
             repo_id=hf_source_repo,
-            ignore_patterns=["model.safetensors", "pytorch_model.bin"],
+            allow_patterns=[
```

Comment:
I'd have preferred an exclude list — but this works for now. We'll have to remember to update this for new model types as needed